apps/web/public/static/locales/en/common.json

```diff
@@ -1234,7 +1235,7 @@
   "theme_dark": "Dark",
   "theme_system": "System default",
   "add_a_team": "Add a team",
-  "saml_config": "SAML Config",
+  "saml_config": "SAML SSO",
```

Comment:
Checked with @PeerRich and agreed the menu item should be called "Single Sign-On".

Reply:
Noted.